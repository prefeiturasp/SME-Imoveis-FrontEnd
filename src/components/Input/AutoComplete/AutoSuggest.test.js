import { mount } from "enzyme";
import React from "react";
import AutoSuggestAddress from "./AutoSuggest";

describe("test <AutoSuggestAddress>", () => {
  let wrapper;
  beforeAll(() => {
    wrapper = mount(<AutoSuggestAddress value="" />);
  });

  it("renders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
