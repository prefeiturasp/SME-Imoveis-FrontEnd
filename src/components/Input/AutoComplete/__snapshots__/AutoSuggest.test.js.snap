// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <AutoSuggestAddress> renders component 1`] = `undefined`;
